server/supernova/audio_backend/portaudio_backend.hpp

```diff
@@ -270,12 +270,12 @@ class portaudio_backend : public detail::audio_backend_base<sample_type, float,
         if (statusFlags & (paInputOverflow | paInputUnderflow | paOutputOverflow | paOutputUnderflow))
             engine_functor::sync_clock();
 
-        const float* inputs[input_channels];
+        const float** inputs = (const float**)alloca(sizeof(float*) * input_channels);
```

Comment:
this is more modernly written as `auto** inputs = static_cast<const float**>(alloca(sizeof(float*) * input_channels));`, same below.